### PR TITLE
redesigned the landing page

### PR DESCRIPTION
**Pull request by Divinejicks**
I am yet to connect the contact us, waitng on matt

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -47,21 +47,26 @@ const config = {
...
+          { to: '/blog', label: 'RESOURCES', position: 'right' },
           {
             type: 'doc',
             docId: 'intro',
-            position: 'left',
-            label: 'Getting started',
+            position: 'right',
+            label: 'FAQs',
+          },
+          { to: '/blog', label: 'CONTACT US', position: 'right' },
```
Comment (deeNewearth) [inline docusaurus.config.js]: Remove INCORRECT links please. Contact us doesn't go to BLOG Once we merge this becomes live, We DONOT want bad links to be indexed. You can comment the code out and Add the links later

File: docusaurus.config.js [hunk 2]
```diff
@@ -112,7 +117,7 @@ const config = {
             ],
           },*/
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} New Earth Art & Tech LLC. Built with Docusaurus.`,
```
Comment (deeNewearth) [inline docusaurus.config.js:115]: Lets remove the Build With Docusaurus

File: docusaurus.config.js [hunk 3]
```diff
@@ -47,21 +47,26 @@ const config = {
...
             docId: 'intro',
-            position: 'left',
-            label: 'Getting started',
+            position: 'right',
+            label: 'FAQs',
+          },
+          { to: '/blog', label: 'CONTACT US', position: 'right' },
+          {
+            to: 'https://rinkeby.staging.colourbox.io/',
+            label: 'GET STARTED',
```
Comment (deeNewearth) [inline docusaurus.config.js]: don't put link to test site, Remove this for now, we will add this when the Live is available

**Review by deeNewearth — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (Divinejicks) [reply docusaurus.config.js]: I have commented out the uncompleted buttons

File: docusaurus.config.js (hunk 2 above)
Comment (deeNewearth) [reply docusaurus.config.js:115]: leave the copyright, just remove "Build With Docusaurus", the line should say Copyright © ${new Date().getFullYear()} New Earth Art & Tech LLC.

**Review by mattygcolourbox — CHANGES_REQUESTED** [review]
Banner image is grainy and has a background color attached to it.

**Review by mattygcolourbox — APPROVED** [review]

**Review by deeNewearth — APPROVED** [review]

File: src/components/landingPage/ColourBoxLandingPage.module.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.container_landing_page {
+    margin: 3rem 20rem;
```
Comment (deeNewearth) [inline src/components/landingPage/ColourBoxLandingPage.module.css:2]: NEVER USE margins to center pages.. margins should only be used for margins

File: src/components/landingPage/ColourBoxLandingPage.module.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.Flex {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    column-gap: 30px;
```
Comment (deeNewearth) [inline src/components/landingPage/ColourBoxLandingPage.module.css:9]: don't mix px and rem

**Pull request merged**